doc/cpp/recipe.rst

```diff
@@ -80,6 +80,17 @@ Recipe
 
         By default returns an empty list.
 
+    .. cpp:function:: virtual std::vector<connection> external_connections_on(cell_gid_type gid) const
```

Comment:
shouldn't this return a `vector<ext_cell_connection>`?

Reply:
Should!